Package.swift

```diff
@@ -172,31 +167,40 @@ let package = Package(
 )
 
 func getGtk4MinorVersion() -> Int? {
+#if os(Windows)
+    guard let pkgConfigPath = ProcessInfo.processInfo.environment["PKG_CONFIG_PATH"],
+          case let tripletRoot = URL(fileURLWithPath: pkgConfigPath, isDirectory: true)
+              .deletingLastPathComponent().deletingLastPathComponent(),
+          case let vcpkgInfoDirectory = tripletRoot.deletingLastPathComponent()
+              .appendingPathComponent("vcpkg").appendingPathComponent("info"),
+          let installedList = try? FileManager.default.contentsOfDirectory(at: vcpkgInfoDirectory, includingPropertiesForKeys: nil)
+            .map({ $0.deletingPathExtension().lastPathComponent }),
+          let packageName = installedList.first(where: { $0.hasPrefix("gtk_") && $0.hasSuffix("_\(tripletRoot.lastPathComponent)") })
```

Comment:
Windows implementation never checks if installed gtk major version is 4 or am I mistaken? 

The Mac/Linux version does gtk4-launch so doesn't need to confirm major version. But as far as I can tell Windows implementation doesn't check and could return minor version of Gtk3 if that is installed instead of 4?

Reply:
In vcpkg "gtk" always refers to Gtk+ 4. We currently rely on vcpkg installation structure for version detection.